charts/app-config/values-integration.yaml

```diff
@@ -2999,9 +2999,7 @@ govukApplications:
       redis:
         enabled: true
         redisUrlOverride:
-          app: &support-redis >
-            redis://shared-redis-govuk.eks.integration.govuk-internal.digital
-          workers: *support-redis
+          workers: redis://shared-redis-govuk.eks.integration.govuk-internal.digital*support-redis
```

Comment:
`*support-redis` typo

Reply:
Thanks!  Was caring more about trying to automate this, rather than looking at what I was copy and pasting 😬.